libs/detectors/x86/detector.py

```diff
@@ -1,3 +1,37 @@
+import json
+import os
+from pathlib import Path
+
+
+def get_or_create_model_json_file(config, device, camera_id):
+    """
+    If no model directory was created for a given device, the default values will be taken from the configuration file
+    and a model_<device>.json file will be created.
+    """
+    base_path = os.path.join(config.get_section_dict("App")["EntityConfigDirectory"], "sources", camera_id)
+    models_directory_path = os.path.join(base_path, "ml_models")
+    json_file_path = os.path.join(models_directory_path, f"model_{device}.json")
+
+    if not os.path.exists(json_file_path):
+        # Hypothesis: source config directory (base_path) should always exists.
+        if not os.path.exists(models_directory_path):
+            Path(models_directory_path).mkdir(parents=True, exist_ok=True)
+
+        # Create .json file
+        json_content = {
+            "model_name": config.get_section_dict("Detector")["Name"],
```

Comment:
This does not allow using different types of detectors per camera. it will take the same Detector parameters for each camera. We need to specify different Detector parameters per camera (and have defaults). So this requires some structure changes in the config files.
The idea is to use this capability to use adapted models for one camera. So for that camera we need to know all the parameters that are currently in the Detector section. However some of them are constant for all models and others will be the same for each adapted camera. The only thing that will be different for each adapted camera is the ModelPath. (But some other parameters will be different from the default values)

If this is confusing, we can chat about it

Reply:
Yes, I believe there are several issues we should tackle if we set the model in the config file, instead of a separate file.
So, first of all, we decided to automatically create the file and, if necessary, modify it manually. However, maybe we can create an endpoint to modify the model of a camera.

Reply:
But this makes sense, right? it's only changing the model path either manually or through and endpoint